src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -44,15 +44,15 @@ public static ShooterSubsystem getInstance(){
     private ShooterSubsystem(){
         topShooter = new CANSparkMax(20, MotorType.kBrushless);
         bottomShooter = new CANSparkMax(21, MotorType.kBrushless);
+        
         shooterElevator = new CANSparkMax(25, MotorType.kBrushless);
```

Comment:
shooterElevator should be removed from the ShooterSubsystem, as well as any elevator related methods. All commands controlling the elevator motor should use the ElevatorSubsystem.

Reply:
done